src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.people;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int NEGATIVE_AGE = -10;
+    public static final int ZERO_AGE = 0;
+    public static final int BELOW_MINIMUM_AGE = 17;
+    public static final int MINIMUM_AGE = 18;
+    public static final int OVER_MINIMUM_AGE = 21;
+    public static final String CORRECT_LOGIN = "qwerty";
+    public static final String INCORRECT_LOGIN = "log";
+    public static final String CORRECT_PASSWORD = "123456";
+    public static final String INCORRECT_PASSWORD = "qwer";
+    public static final String EMPTY_LINE = "";
+
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        people.clear();
+    }
+
+    @Test
+    void register_nullValue_NotOk() {
+        assertThrows(RegistrationException.class, () -> registrationService.register(
```

Comment:
If you use approach to test several not ok cases in one method, you should use different messages to understand which particular case failed. 
I mean, when this test fails, you should see why - it was null login, or null password etc